lib/ecmascript.ts

```diff
@@ -1238,9 +1287,9 @@ export function ToTemporalDateTimeFields(
 
 export function ToTemporalMonthDayFields(
   bag: Temporal.PlainMonthDayLike,
-  fieldNames: readonly (keyof Temporal.PlainMonthDayLike)[]
-) {
-  const entries: [keyof Temporal.PlainMonthDayLike, 0 | undefined][] = [
+  fieldNames: readonly (keyof Temporal.PlainMonthDayLike & PrimitivePropertyNames)[]
+): Required<Temporal.PlainMonthDayLike> {
```

Comment:
`PlainMonthDayLike` includes a `calendar` field. Does `PrepareTemporalFields` return a `calendar` field?

Reply:
It doesn't, but the typing here could have led you to think it did. I made use of the `PrimitiveFieldsOf` type to remove that (as it now better matches the field key inputs).